jira/client.py

```diff
@@ -3403,6 +3395,13 @@ def _add_client_cert_to_session(self):
         client_cert: Tuple[str, str] = self._options["client_cert"]  # to help mypy
         self._session.cert = client_cert
 
+    def _add_ssl_cert_to_session(self):
+        """
+        Adds the client certificate to the request if configured through the constructor.
```

Comment:
Is this adding the client certificate, or the certificate authority?

Reply:
Thanks, this was a copy paste job I failed to update. I've now updated both the name of the function and improved the docstring for clarity.